templates/zerver/plans.html

```diff
@@ -178,6 +178,18 @@ <h2>Enterprise</h2>
         </div>
     </div>
 
+    <div class="discounts-section">
+        <header>
+            <h1>Special Zulip Cloud Standard pricing</h1>
+        </header>
+        <div class="register-buttons">
+            <a href="/for/open-source" class="register-now buttton">Open source <p>Free</p></a>
+            <a href="/for/education" class="register-now button">Education <p>85%+ OFF</p></a>
+            <a href="/for/research" class="register-now button">Research <p>Free</p></a>
+            <a href="/for/events" class="register-now button">Conferences<p>Up to 85% OFF</p></a>
+            <a href="/for/communities" class="register-now button">Non-profits <p>85%+ OFF</p></a>
```

Comment:
Somehow these have different colors; I guess that's because I have clicked some of them?

![image](https://user-images.githubusercontent.com/2746074/126435411-2ebc6a34-8de1-47be-8c6f-4e85208a8313.png)

Reply:
Yeah.

Reply:
Fixed.